hello_world/hello_world.php

```diff
@@ -1,8 +1,2 @@
-<!DOCTYPE html>
-<html>
-  <body>
-    <?php
-    echo "Hello World!";
-    ?>
-  </body>
-</html>
+<?php
+echo 'Hello World';
```

Comment:
enclose a closing tag.

Reply:
This was ommitted by purpose. From the [docs](http://php.net/basic-syntax.instruction-separation):

```
The closing tag of a PHP block at the end of a file is optional, and in some cases omitting it is helpful when using include() or require(), so unwanted whitespace will not occur at the end of files, and you will still be able to add headers to the response later. It is also handy if you use output buffering, and would not like to see added unwanted whitespace at the end of the parts generated by the included files.
```